examples/persistent-storage/cc13x2x7_26x2x7/BUILD.gn

```diff
@@ -28,22 +30,33 @@ project_dir = "${chip_root}/examples/persistent-storage/cc13x2x7_26x2x7"
 
 ti_simplelink_sdk("sdk") {
   include_dirs = [ "${project_dir}/include" ]
-
-  defines = []
-  if (is_debug) {
-    defines += [ "BUILD_RELEASE=0" ]
-  } else {
-    defines += [ "BUILD_RELEASE=1" ]
-  }
+  public_configs = [ ":sysconfig_config" ]
 }
 
 ti_sysconfig("sysconfig") {
   sources = [ "${project_dir}/chip.syscfg" ]
 
   outputs = [
-    "ti_devices_config.c",
+    "ti_radio_config.c",
+    "ti_radio_config.h",
     "ti_drivers_config.c",
     "ti_drivers_config.h",
+    "ti_ble_config.c",
+    "ti_ble_config.h",
+    "ti_dmm_application_policy.c",
+    "ti_dmm_application_policy.h",
+
+    # disabled until upstream generation is aligned
```

Comment:
do we need these commented-out items?

Reply:
They are generated from the sysconfig files, we can probably remove them. It was more of a note if a human comes across them.